README.md

```diff
@@ -51,7 +51,7 @@ source third-party monitor, and use it to collect metrics. Some other monitors
 do not use collectd. However, either type is configured in the same way.
 
 For a list of supported monitors and their configurations,
-see [Monitor Config](./docs/monitor-config.md).
+see [Monitor Config](./docs/monitors/_monitor-config.md).
```

Comment:
What's up with the leading underscore?

Reply:
Previously we had only monitor docs under `monitors`. We had to move `monitor-config.md` to `monitors/` to get this working in product docs. Added a leading underscore since this file is more like a preface to all the other monitor docs.

Reply:
The underscore makes sure this document shows up first in the docs TOC, followed by an alpha list of monitors. We had to make some org changes to work around some limitations between ReadTheDocs and md files.

Reply:
Going to preserve the docs structure in the agent repo. Will handle this requirement outside of the agent.